addons/website_sale/tests/test_website_sale_image.py

```diff
@@ -229,3 +229,35 @@ def test_01_admin_shop_zoom_tour(self):
         self.assertEqual(images[1].image_1920, red_image)
         self.assertEqual(images[2].image_1920, image_gif)
         self.assertEqual(images[3].image_1920, image_svg)
+
+        # CASE: When uploading a product variant image
```

Comment:
Since you choose to make the "complex" version of the method with splitting the 2 super, you can merge to 2 tests into one: call `create` on `product.image` only once, but with an array of vals.

This will test the 2 basic cases and the split of super at the same time.